servicecontrol/plugins.md

```diff
@@ -112,11 +116,19 @@ The result of a custom check is either success or a failure (with a detailed des
 
 - [How to Develop Custom Checks for ServicePulse](/servicepulse/how-to-develop-custom-checks.md)
 
+#### Should I deploy the CustomChecks plugin to Production?
+Please ensure that you **do deploy** this plugin to your endpoint in production, as it sends relevant information that the developers have specifically coded to highlight as problems to the Ops department to take notice of and act when something fails. This information is displayed in the ServicePulse dashboard.
+
+
 ### ServiceControl SagaAudit Plugin
```

Comment:
Reword to "Saga trace plugin" to make it more obvious that this isn't audit? (yes we should rename the actual plugin as well as a separate task, `Particular.Workflow.Tracing` or similar)